### PR TITLE
Added Alert Box Component

### PR DESCRIPTION
**Pull request by vistajess**

### CODE REVIEW

**Comment by srph** [comment]
Screenshot, please :smile:

**Comment by vistajess** [comment]
alert

**Comment by srph** [comment]
IMO, It's better to name it as `alert` instead of `alert-box`.

File: styles/components/alert.scss [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+	&.-info {
+		background-color : $brand-blue;
+	}
+	&.-warning {
+		background-color : $brand-sun;
+	}
+	&.-danger {
+		background-color: $brand-crimson;
+	}
+	.dismiss-btn {
```
Comment (srph) [inline styles/components/alert.scss]: Exclude this for now. I don't think this belongs to alert.

File: styles/components/alert.scss [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+		background: transparent;
+		color: $brand-white;
+    border: 1px solid $brand-white;
+    width: 20px;
+    height: 20px;
+    text-align: center;
+    line-height: 16px;
+    cursor: pointer;
+    border-radius: $border-radius-rounded;
+   	-webkit-transition: background,color 0.3s linear;
```
Comment (srph) [inline styles/components/alert.scss]: Leave out vendor prefixes. We'll use [autoprefixer](https://github.com/postcss/autoprefixer) to do this for us (automation) on release.

File: styles/components/alert.scss [hunk 3]
```diff
@@ -1,3 +1,40 @@
 .alert {
+	padding: 20px 40px;
+	margin: 10px;
+	position: relative;
+	color: $brand-white;
 
+	.dismiss-btn {
```
Comment (srph) [inline styles/components/alert.scss]: I don't think this belongs to `alert`. You may remove this ( as well as the example ) .

File: styles/components/alert.scss [hunk 4]
```diff
@@ -1,3 +1,40 @@
 .alert {
+	padding: 20px 40px;
+	margin: 10px;
```
Comment (srph) [inline styles/components/alert.scss]: No margins! To put margins on the examples, add `doc-bottom-space` or `doc-bottom-space-large`.

File: styles/components/alert.scss [hunk 5]
```diff
@@ -1,3 +1,40 @@
 .alert {
+	padding: 20px 40px;
```
Comment (srph) [inline styles/components/alert.scss]: Put smaller values :smile:. Our alert box is too big!

File: styles/components/alert.scss (hunk 4 above)
Comment (vistajess) [inline styles/components/alert.scss]: Do we have this class doc-bottom-space?

Comment (srph) [inline styles/components/alert.scss]: Yea, for docs only. Check https://github.com/dev-onenetworkecommerce/june/pull/21. Try pulling from the `master` branch.

File: styles/components/alert.scss [hunk 6]
```diff
@@ -1,3 +1,19 @@
 .alert {
+  padding: 10px 20px;
+	margin: 10px;
```
Comment (srph) [inline styles/components/alert.scss]: Remove margin :smile:

File: docs/sections/Docs-Alerts/index.jsx [hunk 7]
```diff
@@ -4,6 +4,24 @@ export default class Alerts extends React.Component {
...
+      	</div>
+      	<div className="alert -info">
+      	  <span>Class: alert-box -info</span>
+      		<h3>Heads up! This alert needs your attention, but its not super important.</h3>
+      	</div>
+      	<div className="alert -warning">
+      	  <span>Class: alert-box -warning</span>
+      		<h3>Warning! Better check yourself, youre not looking too good.</h3>
+      	</div>
+      	<div className="alert -danger">
```
Comment (srph) [inline docs/sections/Docs-Alerts/index.jsx:21]: You can add `.doc-bottom-space` for the margins

**Comment by srph** [comment]
Almost good to go! Good job :+1:

**Comment by vistajess** [comment]
Kindly check if its ok. :)

File: styles/components/alert.scss [hunk 8]
```diff
@@ -1,3 +1,36 @@
+$def-margin : 5px !default;
```
Comment (srph) [inline styles/components/alert.scss]: I asked you to remove the margins hahahaha

File: styles/components/alert.scss [hunk 9]
```diff
@@ -1,3 +1,36 @@
...
+$large-margin : 15px;
```
Comment (srph) [inline styles/components/alert.scss]: :smile:

File: styles/components/alert.scss [hunk 10]
```diff
@@ -1,3 +1,36 @@
...
+@mixin margin($position,$px) {
```
Comment (srph) [inline styles/components/alert.scss]: Noooo

File: styles/components/alert.scss [hunk 11]
```diff
@@ -1,3 +1,36 @@
...
+  background-color : $brand-blue;
+}
+.-warning {
+  background-color : $brand-sun;
+}
+.-danger {
+  background-color: $brand-crimson;
+}
 
+.doc-top-space {
```
Comment (srph) [inline styles/components/alert.scss]: May ganto na tangeks. Pull ka lang sa master hahah :laughing:

File: docs/sections/Docs-Alerts/index.jsx [hunk 12]
```diff
@@ -4,6 +4,24 @@ export default class Alerts extends React.Component {
   render() {
     return (
       <div>
+        <h1>Alert</h1>
+      	<strong>This is a collection of symbols for quickly mocking up web applications. Created by your friends at One Network Ecommerce and released under the ONE License</strong>
+      	<div className="alert -success doc-top-space">
```
Comment (srph) [inline docs/sections/Docs-Alerts/index.jsx]: `doc-bottom-space` lang. Let's avoid `top` _margin_s

File: styles/components/alert.scss (hunk 11 above)
Comment (vistajess) [inline styles/components/alert.scss]: Hahahah meron na ba :D di kc ako nagpull hahah.. sareeh

File: docs/sections/Docs-Alerts/index.jsx (hunk 12 above)
Comment (vistajess) [inline docs/sections/Docs-Alerts/index.jsx]: Ok  :+1:

**Pull request merged**